crawlers/deploy/polygon-state.service

```diff
@@ -8,6 +8,6 @@ User=ubuntu
 Group=www-data
 WorkingDirectory=/home/ubuntu/moonstream/crawlers/mooncrawl
 EnvironmentFile=/home/ubuntu/moonstream-secrets/app.env
-ExecStart=/home/ubuntu/moonstream-env/bin/python -m mooncrawl.state_crawler.cli --access-id "${NB_CONTROLLER_ACCESS_ID}" crawl-jobs --blockchain polygon
+ExecStart=/home/ubuntu/moonstream-env/bin/python -m mooncrawl.state_crawler.cli --access-id "${NB_CONTROLLER_ACCESS_ID}" crawl-jobs --blockchain polygon --infura --jobs-file ./mooncrawl/state_crawler/jobs/polygon-jobs.json
```

Comment:
`--jobs-file ./mooncrawl/state_crawler/jobs/polygon-jobs.json`
to
`--jobs-file /home/ubuntu/moonstream/crawlers/mooncrawl/state_crawler/jobs/polygon-jobs.json`

Reply:
better to use full path in systemd files

Reply:
or env variable